resources/ext/ezbake.conf

```diff
@@ -14,6 +29,10 @@ ezbake: {
              }
 
       debian: { dependencies: ["puppet-agent"],
+               # see redhat comments on why this is terrible
```

Comment:
We don't want any of it, these were hacks that should be removed and then fixed properly, e.g. in SERVER-358 and other related tickets.